Remove Moped support (#842)
appsignal/appsignal-ruby: .changesets/remove-moped-support.md

---

> bump: "patch"

Reviewer: If we're dropping support for something, maybe it should not be a `patch` release? @tom thoughts?

Reviewer 2: It should be at least a minor release like Jeff says. I wouldn't want to make this a major version release.

Author: https://docs.appsignal.com/ruby/integrations/mongodb.html#older-mongoid-versions-and-mongomapper I think we virtually removed Moped support in the past. This PR removes the last trace of it left; that's why I chose to go with a patch.

type: "remove"
---

> Remove Moped support as it is no longer the official ruby Mongo driver and it's been 7 years
> without mainteinance.

Reviewer: ```suggestion
Remove Moped support as it is no longer the official Ruby Mongo driver and it's been unmaintained for 7 years.
```
